### DISCUSSION TITLE
请帮忙恢复专业版，我的邮箱收不到验证码

### DISCUSSION THREAD

**Discussion by yanggaojun101** [getlantern/lantern-forum-cn#357] [Q&A]
邮箱是 yanggaojun101@126.com
![b1c25a8f-bf61-4cca-8640-de67fda37cc1](https://github.com/getlantern/lantern-forum-cn/assets/3292215/13fdb663-8631-4d30-b6b1-04fdb94989f6)

**Comment by yanggaojun101** [comment]
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@yanggaojun101 嗯嗯. 好的. 我来帮您问看. 不好意思啊, 要等晚上团队上线了才能帮您解决了.

**Comment by MaxMKMax** [comment]
@yanggaojun101 对了, 有用户能用邮箱登陆来解决这个问题. 你能试试看嘛?

**Reply by yanggaojun101** [reply]
邮箱登录是哪里，就是邮箱收验证码的地方吗？我收不到

**Reply by MaxMKMax** [reply]
@yanggaojun101 就是用邮箱登陆. 然后应该会收到邮件什么的.

如果不可以也没事. 我已经帮你问着付款团队了.

**Reply by yanggaojun101** [reply]
是通过“账号”--“授权设备使用专业版”--“通过邮箱授权登录”吗？
我通过邮箱收不到验证码

**Comment by yanggaojun101** [comment]
@Mike777Khoo 一直收不到验证码，我用新的邮箱 [yanggaojun101@gmail.com](mailto:yanggaojun101@gmail.com)重新买了一个新的。我想请问，原先的[yanggaojun101@126.com](mailto:yanggaojun101@126.com)的剩余时间能转到我的gmail账号来吗？

![30593a0a-8ac8-4a73-8fcc-d52de09a9fef](https://github.com/getlantern/lantern-forum-cn/assets/3292215/c238983d-668a-400b-ad19-af2713440461)
